src/common/datetime.cpp

```diff
@@ -541,6 +541,52 @@ bool wxDateTime::IsLeapYear(int year, wxDateTime::Calendar cal)
     }
 }
 
+#ifdef _WIN32
+#include <wx/msw/registry.h>
+
+/* static */
+wxDateTime::WeekDay wxDateTime::GetFirstWeekDay(bool *success) {
+    wxRegKey key("HKEY_CURRENT_USER\\Control Panel\\International");
+    wxString val;
+
+    if (key.Exists() && key.HasValue("iFirstDayOfWeek")) {
+        key.QueryValue("iFirstDayOfWeek", val);
+        if (success)
+             *success = true;
+        return wxDateTime::WeekDay((wxAtoi(val) + 1) % 7);
+	} else {
+        if (success)
+             *success = false;
+        return wxDateTime::Sun;
+	}
+}
+
+#elif defined(__APPLE__)
+// implementation in utils_base.mm
+#elif defined(__linux__)
+
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE
```

Comment:
Is this really needed? It's supposed to be already defined by configure normally...